### PR TITLE
#1431 Add missing translations

### LINKED ISSUE #1431: Add missing translations
### Describe the problem

The following problems were reported by users:

### To reproduce

Open 2.0.0 IDE2, and set the language to non-English.

### Expected behavior

All text defined by IDE2/Theia must be translated when a translation exists.

----

> All text defined by IDE2/Theia

Not from the CLI.

### Arduino IDE version

2.0.0

### Operating system

N/A

### Operating system version

n/a

### Additional context

Go through the code and add the missing translations.

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the problem still occurs when using the latest [nightly build](https://github.com/arduino/arduino-ide#nightly-builds)
- [X] My report contains all necessary details

### PR DESCRIPTION
**Pull request by kittaakos**
### Motivation

Add the missing translations.

Was fixed:
 - `File` > `Save` and `File` > `Save As..`:
    Screen Shot 2022-09-19 at 13 40 47
 - `Edit`, `Edit` > `Copy`, and `Edit` > `Paste`:
    Screen Shot 2022-09-19 at 13 41 14
 - `Help` menu:
    Screen Shot 2022-09-19 at 13 48 18

 - _Search in workspace_: `Replace`, `files to include`, and `files to exclude` labels and all `Match Case`, `Match Whole Word`, etc tooltips:

    https://user-images.githubusercontent.com/1405703/191009653-6447aa29-0263-4d12-9edb-1c593aba4c13.mp4
 - Tooltips for the _Terminal_:
    Screen Shot 2022-09-19 at 13 46 49

 - `Install`/`Uninstall` in the library/boards manager:

    https://user-images.githubusercontent.com/1405703/191009891-94fe7552-8241-46d9-b5ae-03750cf1bc0c.mp4
 - `Tools` > `Serial ports` and `Tools` > `Network ports` (no translations yet):
 - `Settings` > `Network` > `Manual Proxy Configuration` (no translations yet):
 - `Boards`, `Ports`, and `Show all ports` in the board select dialog (no translations yet):
 - `Couldn't open serial plotter` (no translations yet):
 - Translated compiler warnings in `Settings` dialog: `All`, `None`, `Default`, and `More` (no translations yet):
 - `Find`/`Replace` widget in editors + all Monaco-based language contributions:
    Screen Shot 2022-09-19 at 17 02 27

Language pack changes:
 - Downgraded the following language packs to `1.53.2`: 
     - French (`fr`),
     - Chinese (Simplified) (`zh-hans`),
     - German (`de`),
     - Japanese (`ja`),
     - Turkish (`tr`),
     - Italian (`it`),
     - Russian (`ru`),
     - Spanish (`es`),
     - Portuguese (Brazil) (`pt-BR`), and
     - Czech (`cs`).
 - Added the following language packs: 
     - Hungarian (`hu`) [`1.48.3`],
     - Bulgarian (`bg`) [`1.48.3`],
     - Polish (`id`) [`1.53.2`],
     - Ukrainian (`uk`) [`1.48.3`], and
     - Korean (`ko`) [`1.53.2`].

WONT FIX:
 - `Hide` in the view-container context menu. (Sorce: https://github.com/arduino/arduino-ide/issues/1431#issuecomment-1246905029, upstream issue: https://github.com/eclipse-theia/theia/issues/11681) 
> Context menu item in view containers, such as _Debug_ view:
> Screen Shot 2022-09-14 at 17 29 03

### Other information

Closes #1431

### Reviewer checklist

* [ ] PR addresses a single concern.
* [ ] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-ide/pulls) before creating one)
* [ ] PR title and description are properly filled.
* [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by AlbyIanna — APPROVED** [review]
Code looks good to me ✅

**Comment by kittaakos** [comment]
This PR should contain #1462.

File: arduino-ide-extension/src/node/i18n/arduino-localization-contribution.ts [hunk 1]
```diff
@@ -3,150 +3,45 @@ import {
...
+    ['es'],
+    ['fr'],
+    ['hu'],
+    // ['id'], Does not have Transifex translations, but has a VS Code language pack available on Open VSX.
+    ['it'],
+    ['ja'],
+    ['ko'],
+    ['nl'],
+    ['pt-br', 'pt'],
+    ['pl', 'pl'],
```
Comment (per1234) [inline arduino-ide-extension/src/node/i18n/arduino-localization-contribution.ts:29]:
```suggestion
    ['pl'],
    ['pt-br', 'pt'],
```

- Use alphabetical order as specified in the comment
- Remove superfluous second element for Polish

**Review by per1234 — CHANGES_REQUESTED** [review]

**Review by per1234 — APPROVED** [review]
Approved code changes and behaviour. Thanks Akos!

**Pull request merged**